### PR TITLE
Fix description empty string creating bug in layout

### PR DESCRIPTION
**Pull request by ildyria**

### CODE REVIEW

**Review by d7415 — APPROVED** [review]

**Comment by Gendra13** [comment]
Thank you. 
I did test the fix and it works in my special case (no description at all), but there is still an issue when there is a description text present. 
I added a bit more information in the corresponding report: https://github.com/LycheeOrg/Lychee/discussions/2399#discussioncomment-9178818

**Comment by ildyria** [comment]
@Gendra13  should be fixed now. I found the culprit.

**Comment by Gendra13** [comment]
Yes, thank you. That fix work fine.

File: resources/views/components/gallery/album/details.blade.php [hunk 1]
```diff
@@ -38,7 +38,7 @@
     @endif
 </div>
 @if($this->AlbumFormatted->description !== '')
-<div class="w-full px-7 my-4 text-justify text-text-main-200 prose prose-invert prose-sm" {{ $attributes }} >
+<div class="w-full px-7 max-w-full my-4 text-justify text-text-main-200 prose prose-invert prose-sm" {{ $attributes }} >{{-- prose set max-width to 65 chars --}}
```
Comment (d7415) [inline resources/views/components/gallery/album/details.blade.php:41]: Do we need w-full *and* max-w-full?

**Review by d7415 — APPROVED** [review]

File: resources/views/components/gallery/album/details.blade.php (hunk 1 above)
Comment (ildyria) [reply resources/views/components/gallery/album/details.blade.php:41]: yes.
`w-full` = `width: 100%`
`max-w-full` = `max-width: 100%`

**Pull request merged**